nw-firestore-schema.ts

```diff
@@ -101,7 +101,17 @@ interface FAQ {
   lastModifiedBy: string;
 }
 
+interface CMS {
+  CMSOff: boolean;
+  OffUntilDate: timestamp;
+}
+
+interface InternalWebsites {
```

Comment:
is this/should this be a collection?

Reply:
I think its fine for it to be a collection, we might want to add feature flags for our other internal projects too.

Reply:
Yeah I wanted it to be a collection

Reply:
it is a collection

Reply:
oh, I thought it would have a structure more similar to `Hackathons` which is defined as ` Hackathon[]`

Reply:
I'm not really sure what that structure will look like yet so right now I just chose to make one for the CMS